docs/tables.txt

```diff
@@ -1,5 +1,147 @@
-.. _tables:
+.. _tables-browser:
 
 ==============
 Tables Browser
 ==============
+
+The CrateDB admin UI comes with a *tables browser* that allows you to inspect
+and query regular `document tables`_ as well as `BLOB tables`_.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Screenshots
+===========
+
+Document Tables
+---------------
+
+Here's what a simple document table looks like:
+
+.. image:: tables-simple-1.png
+
+Here you can see a number of :ref:`table-level datapoints and settings are
+<table-level-info>` listed.
+
+Below this, you will find information about the table :ref:`schema
+<schema-info>`:
+
+.. image:: tables-simple-2.png
+
+BLOB Tables
+-----------
+
+If you have BLOB tables, you can access them by selecting the *Blob Tables* menu
+item on the left-hand sub-navigation menu:
+
+.. image:: tables-blobs-1.png
+
+Like document tables, a number of :ref:`table-level datapoints and settings
+<table-level-info>` are listed first, and then below that, you will find
+information about the table :ref:`schema <schema-info>`:
+
+.. image:: tables-blobs-2.png
+
+Features
+========
+
+.. |nbsp| unicode:: 0xA0
+   :trim:
+
+.. _table-level-info:
+
+Table-Level Information
+-----------------------
+
+The table overview displays the following table-level information:
+
+:Name:
+
+  The name of the table.
+
+:Health:
+
+  The health of the table.
+
+  FIXME: what are the available options
```

Comment:
@mxm here

Reply:
This is red, yellow, or green.

>The cluster health status is: green, yellow or red. On the shard level, a red status indicates that the specific shard is not allocated in the cluster, yellow means that the primary shard is allocated but replicas are not, and green means that all shards are allocated. The index level status is controlled by the worst shard status. The cluster status is controlled by the worst index status.

https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html